### ISSUE TITLE
[Feature request] Increase height of the splitter bar of "Search results" bottom pane

### ISSUE THREAD

**Issue by vlakoff** [notepad-plus-plus/notepad-plus-plus#15980]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Description of the Issue

I'm a heavy user of the "Search results" bottom pane. Sometimes it's for a single code file, sometimes it's for an entire code project, and sometimes it's for large searches across several drives.

Depending on the use case, sometimes I want a small "Search results" bottom pane (so that there is space to see the opened file), and sometimes a large "Search results" bottom pane is very appreciated (so that the search results are more convenient and efficient to navigate).

Therefore, for optimal use, I often need to resize this pane. But here is the problem: the clickable area of the splitter bar is very small. It requires very precise use of the mouse, which is frustrating each time, and even very difficult to achieve when drunk, as I am currently.

### Describe the solution you'd like.

A welcome improvement would be to increase the height of this bar, and thus its clickable area.

### Debug Information

```shell
Notepad++ v8.6.7
```

**Comment by xomx** [comment]
![Image](https://github.com/user-attachments/assets/79255379-5d76-4c05-ae9c-20394a5a3f8b)

![Image](https://github.com/user-attachments/assets/2e9e6eda-1d45-4681-9152-268676c6de43)

@donho 
This issue belongs to the ongoing longterm hidpi stuff.
I should be able to fix that, please assign me here, I need a bigger splitter too.

**Comment by vlakoff** [comment]
Indeed, I use a 4K monitor with 150% scaling, and the draggable area has a height of just 4 pixels… which is remarkably small on a 4K screen.
